README.md

```diff
@@ -1,29 +1,46 @@
-# MODX Revolution
+<p align="center">
+  <a href="https://modx.com/">
+    <img alt="MODX Revolution" src="https://modx.com/assets/design/css/images/modx-logo-color.svg" width="180" />
+  </a>
+</p>
+<h1 align="center">
+  MODX Revolution
+</h1>
 
-[![Build Status](https://travis-ci.org/modxcms/revolution.svg?branch=2.x)](https://travis-ci.org/modxcms/revolution) [![Slack Chat](https://img.shields.io/badge/chat_in_slack-online-green.svg?longCache=true&style=flat&logo=slack)](https://modx.org)
+#### MODX Revolution is the world’s fastest, most secure, flexible and scalable Open Source CMS.
+
+
+[![LICENSE](https://img.shields.io/badge/License-GPL%20v2-blue.svg)](./LICENSE) [![Build Status](https://travis-ci.org/modxcms/revolution.svg?branch=2.x)](https://travis-ci.org/modxcms/revolution) [![Contributors](https://img.shields.io/github/contributors/modxcms/revolution.svg)](https://github.com/modxcms/revolution/graphs/contributors) [![Slack Chat](https://img.shields.io/badge/chat_in_slack-online-green.svg?longCache=true&style=flat&logo=slack)](https://modx.org) [![follow on Twitter](https://img.shields.io/twitter/follow/modx.svg?style=social&logo=twitter)](https://twitter.com/intent/follow?screen_name=modx)
 
 ## Content Management System and Application Framework
 
 MODX lets you power anything from multi-language, multi-domain corporate sites to personal blogs to mobile APIs. Delivering true creative freedom and removing all restrictions, it lets you control the markup and design without having to code. You can also tailor its modular and extensible core to accommodate virtually any custom requirement or amount of traffic. MODX is the free open source software that meets your needs today—and tomorrow.
 
+MODX Revolution is the world’s fastest, most customizable Open Source PHP CMS. Your creative vision, no restrictions, no compromise.
+
+-   **50× Fewer Vulnerabilities** in government security databases
+-   **Millions** of Downloads
+-   **78 Versions Released** for relentless improvement
+-   Over **800** add-ons, **over 6 600 000** downloads, and an **uncountable** number of happy MODX-ers.
```

Comment:
I don't think we should include information like statistics in the README. They change and require to many unnecessary changes over time.

Reply:
Agree